### PR TITLE
Enhancement: Add issue template for bug report

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] provides a GitHub issue template for a bug report

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Affected System
+
+#### `php -v`
+
+```txt
+<!-- please replace this with the output of running `php -v` --> 
+```
+
+#### `composer show | grep 'fzaninotto/faker'`
```
Comment (pimjansen) [inline .github/ISSUE_TEMPLATE/bug.md]: I would just add the description on whats needed here:
`#### Faker version`

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+name: Bug
+about: Something does not work as expected
+----
+
+### Summary
+
+<!-- provide a summary here -->
```
Comment (pimjansen) [inline .github/ISSUE_TEMPLATE/bug.md]: Somehow i dont see this in the markdown preview, is that just a `github` thing or do we need to push it in a code block?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Affected System
+
```
Comment (pimjansen) [inline .github/ISSUE_TEMPLATE/bug.md]: Maybe we should explain them atleast what is expected here like in the other blocks as well

File: .github/ISSUE_TEMPLATE/bug.md (hunk 2 above)
Comment (localheinz) [reply .github/ISSUE_TEMPLATE/bug.md]: It's rendered as an HTML comment!

**Pull request merged**